lib/verifier.ts

```diff
@@ -12,8 +12,21 @@ const claims = [
   Claims.AUD,
 ];
 
-const parseToNumber = (value?: string) =>
-  value ? parseInt(value) || null : null;
+const parseToNumber = (value?: string | number) => {
+  if(typeof(value) === 'number') {
+    return value;
+  } else if(typeof(value) === 'string') {
+    const parsed = parseInt(value);
+
+    if (isNaN(parsed)) {
+      return null;
```

Comment:
`0` is falesy in classic JS for a reason. As you mentioned the numbered claims should never be `0`.

The claims functions must be able to handle strings for pure JS implementations of this library.

Since you have defined the `JWTStandardClaims` as optional props though (without mentioning null), it would be better to return `undefined` here instead of `null`. Also, leaving the `0` return as undefined will be less of a potential breaking change.

I would simplify this to:

```ts
const parseToNumber = (value?: string | number) =>
  value ? parseInt(value.toString()) || undefined : undefined;
```